ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -328,6 +328,12 @@ tests:
   steps:
     cluster_profile: kubevirt
     workflow: openshift-e2e-kubevirt
+- as: e2e-aws-single-node-multi
```

Comment:
Not sure about the name. "Single node multi" is confusing
Why not removing the old one?

Reply:
Didn't want to remove anything until I'm confident this new approach works well. I guess that's not a good enough reason. Will remove the old one and change this name